tools/codegen/model.py

```diff
@@ -1058,10 +1079,14 @@ def strip_ret_annotation(r: Return) -> Return:
                 annotation=None,
             )
 
+        base_name = self.name.name.base
+        if strip_view_copy_name and base_name.endswith('_copy'):
+            base_name = base_name.replace('_copy', '')
```

Comment:
If `_copy` is semantically meaningful in this way it would feel better to represent it in the name structure (in the same we represent inplace-ness and dunder-ness).

Reply:
yeah i was a little torn about this - I don't think a trailing `_copy` in the name is semantically meaningful on its own. It's only meaningful if we also spot another `NativeFunction` that's a view op, with the same arguments and name but without the trailing `_copy`.

This function (`signature(strip_view_copy_name=True)`) will end up returning the same `FunctionSchema` object for  `{op}` and `{op}_copy`, but in the logic to group `NativeFunctionViewGroup` objects, we only actually do the grouping if the first op is a view op.

Reply:
okey dokey